bin/docker-entrypoint

```diff
@@ -20,6 +20,16 @@ scheduler() {
   exec /usr/local/bin/celery worker --app=redash.worker --beat -s$SCHEDULE_DB -c$WORKERS_COUNT -Q$QUEUES -linfo --max-tasks-per-child=10 -Ofair
 }
 
+dev_scheduler() {
+  WORKERS_COUNT=${WORKERS_COUNT:-1}
+  QUEUES=${QUEUES:-celery}
```

Comment:
As we have a dedicated dev worker entrypoint, we can have defaults that are inline with it:

WORKERS_COUNT = 2
QUEUES = "queries,scheduled_queries,celery,schemas"

Reply:
57e51a5